src/main/kotlin/com/msd/robot/application/RobotController.kt

```diff
@@ -25,9 +26,19 @@ class RobotController(
     @PostMapping
     fun spawnRobot(@RequestBody spawnDto: RobotSpawnDto): ResponseEntity<RobotDto> {
         val robot = robotApplicationService.spawn(spawnDto.player, spawnDto.planet)
-        return ResponseEntity.ok(robotMapper.robotToRobotDto(robot))
+        return ResponseEntity.status(HttpStatus.CREATED).body(robotMapper.robotToRobotDto(robot))
     }
 
+//    @PostMapping
```

Comment:
Why did you already add this and then commented it out, if the new api hasn't been defined yet?

Reply:
I wanted to try out if it is as easy as we thought, but then realized that we cannot use the same endpoint twice. But now that it's there for a future implementation we can just modify this stub